src/Etoc/NewsLetter.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace eLife\Journal\Etoc;
+
+abstract class NewsLetter
```

Comment:
I think name is meant to match `Subscription::getNewsletters()`, which is Civi vocabulary. Lowercase `l` though?

Reply:
Correcting myself: this is our vocabulary, not Civi. Same comment applies for consistency.